modules/030-cloud-provider-azure/hooks/wait_for_all_master_nodes_to_become_ready.go

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
we have library [hook](https://github.com/deckhouse/deckhouse/blob/main/go_lib/hooks/nodes/hook.go#L34) for it

Can we use it here?